context/app/static/js/components/detailPage/provenance/ProvTable/ProvTable.tsx

```diff
@@ -40,56 +41,75 @@ function ProvEntityColumn({
 
   const displayMissingAncestors =
     missingAncestors && missingAncestors.length > 0 && entities.length === 0 && type !== 'Dataset';
+
+  const noDisplayedContent = entities.length === 0 && !displayMissingAncestors && !descendantEntityCounts?.[type];
+
+  if (noDisplayedContent) {
+    return (
+      <Alert severity="warning" $width="100%">
+        No {type.toLowerCase()}s available.
+      </Alert>
+    );
+  }
+
+  return (
+    <>
+      {entities.length > 0 &&
+        entities
+          .sort((a, b) => a.created_timestamp - b.created_timestamp)
+          .map((item, j, items) => {
+            const isSampleSibling =
+              j > 0 && item.entity_type === 'Sample' && items[j - 1]?.sample_category === item.sample_category;
+            if (isSampleSibling && !isExpanded[item.sample_category as string]) {
+              const siblings = items.filter((i) => i.sample_category === item.sample_category);
+              const numberOfSiblings = siblings.length - 1;
+              // Only draw the button once for the first sibling in the list
+              // First element in the siblings list is the first item in the category,
+              // so the second item in the list is the first sibling.
+              const isFirstSibling = siblings.findIndex((i) => i.uuid === item.uuid) === 1;
+              if (!isFirstSibling) return null;
+              return (
+                <Button
+                  key={item.uuid}
+                  variant="contained"
+                  onClick={() => setIsExpanded((s) => ({ ...s, [item.sample_category as string]: true }))}
+                >
+                  View More in Category ({numberOfSiblings})
+                </Button>
+              );
+            }
+            return (
+              <ProvTableTile
+                key={item.uuid}
+                uuid={item.uuid}
+                id={item.hubmap_id}
+                entity_type={item.entity_type}
+                isCurrentEntity={currentEntityUUID === item.uuid}
+                isSampleSibling={isSampleSibling}
+                isFirstTile={j === 0}
+                isLastTile={j === type.length - 1}
+                onClick={() =>
+                  trackEntityPageEvent({ action: 'Provenance / Table / Select Card', label: item.hubmap_id })
+                }
```

Comment:
Tiny optimization - we might as well take the chance to pull this function out of the `.map` call and use a `useCallback` up above; this way, only one instance of this function is created/shared by all the tiles instead of creating separate onClick functions for each tile:

```
const trackProvTableTile = useCallback(() => trackEntityPageEvent({ action: 'Provenance / Table / Select Card', label: item.hubmap_id }), [trackEntityPageEvent]}
```